adagucserverEC/CDPPGoes16Metadata.cpp

```diff
@@ -20,7 +20,7 @@ int CDPPGoes16Metadata::isApplicable(CServerConfig::XMLE_DataPostProc *proc, CDa
 
 int CDPPGoes16Metadata::execute(CServerConfig::XMLE_DataPostProc *proc, CDataSource *dataSource, int mode) {
   CDBDebug("execute");
-  if ((isApplicable(proc, dataSource) & mode) == false) {
+  if ((isApplicable(proc, dataSource, mode) & mode) == false) {
```

Comment:
So this checks if the requested mode `mode` is compatible with the mode that te postprocessor supports (returned by isApplicable), correct?

In `CDPPOperator` support is dependent on the requested mode, so you had to add this to `isApplicable` function?

Reply:
Will move `& mode` part into the method.